R/expect_lint.R

```diff
@@ -41,10 +41,12 @@
 #' @export
 expect_lint <- function(content, checks, ..., file = NULL, language = "en") {
   if (!requireNamespace("testthat", quietly = TRUE)) {
-    stop( # nocov start
-      "'expect_lint' is designed to work within the 'testthat' testing framework, but 'testthat' is not installed.",
-      call. = FALSE
-    ) # nocov end
+    # nocov start
+    cli_abort(c(
+      i = "{.fun expect_lint} is designed to work within the {.pkg testthat} testing framework.",
+      x = "{.pkg testthat} is not installed."
+    ))
+    # nocov end
```

Comment:
this message is way too verbose IMO. R/addins.R is much better: `{.pkg testthat} is required to use {.fun expect_lint}`. The old message was OK because it's just a sentence, I think it's a lot harder to process all the visual elements introduced by {cli}:

```
Error:
ℹ `expect_lint()` is designed to work within the testthat testing
  framework.
✖ testthat is not installed.
Run `rlang::last_trace()` to see where the error occurred.
```

vs

```
Error: 'expect_lint' is designed to work within the 'testthat' testing framework, but 'testthat' is not installed.
```